.coveragerc

```diff
@@ -7,6 +7,8 @@ omit =
     urllib3/contrib/appengine.py
     urllib3/contrib/ntlmpool.py
     urllib3/contrib/pyopenssl.py
+    urllib3/contrib/securetransport.py
```

Comment:
Yup. And yet, if we want full coverage for this we'll probably get near to doubling the amount of test code in urllib3 because we have to monkeypatch *every single macOS function* to fail in its various valid possible ways to exercise the relevant code paths. That's a very substantial undertaking, and it's hard to justify only doing it for securetransport and not pyopenssl. Ultimately, these modules are considered "best effort" unless someone is willing to step in and write all the test code that would turn them into productised code.